Package.swift

```diff
@@ -8,22 +8,18 @@ let package = Package(
     products: [
         .library(
             name: "Subprocess",
-            type: .static,
             targets: [ "Subprocess" ]
         ),
         .library(
             name: "SubprocessMocks",
-            type: .static,
             targets: [ "SubprocessMocks" ]
         ),
         .library(
             name: "libSubprocess",
```

Comment:
There is no need for this whole `.library` product if the `type` is removed.  It's identical to the `Subprocess` product.  Same goes for `libSubprocessMocks`; it does not need to exist if the `type` is removed.

Obviously removing entire products will be a breaking change and would require a major version update.

Perhaps removing the `type` first with this PR, and tag this as a 3.x release.  Then a separate PR removing the duplicated products which can be tagged as a 4.0 release would be the right path forward.

Reply:
Sounds good, so no other actionable items other than perhaps writing down that the duplicated products type needs to be removed? I could add it as a comment under the issue https://github.com/jamf/Subprocess/issues/29?